ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/l2-protocol-forwarding.j2

```diff
@@ -0,0 +1,33 @@
+{# doc - l2 protocol forwarding #}
+{% if l2_protocol.forwarding_profiles | arista.avd.default([]) | length > 0 %}
+
+## L2 Protocol Forwarding
+
+### Forwarding Profiles Summary
+
+| Profile | BFD Per-link RFC-7130 | E-LMI | ISIS | LACP | LLDP | MACSEC | Pause | STP |
+| ------- | --------------------- | ----- | ---- | ---- | ---- | ------ | ----- | --- |
+{%     for profile in l2_protocol.forwarding_profiles | arista.avd.natural_sort('name') %}
+{%         set rows = {} %}
+{%         for protocol in profile.protocols %}
+{%             set proto_name = protocol.name | replace(" ", "_") | replace("-", "_") %}
+{%             do rows.update({proto_name: []}) %}
+{%             if protocol.forward is arista.avd.defined(true) %}
+{%                 do rows[proto_name].append('forward') %}
+{%             endif %}
+{%             if protocol.tagged_forward is arista.avd.defined(true) %}
+{%                 do rows[proto_name].append('tagged forward') %}
+{%             endif %}
+{%             if protocol.untagged_forward is arista.avd.defined(true) %}
+{%                 do rows[proto_name].append('untagged forward') %}
+{%             endif %}
+{%         endfor %}
+| {{ profile.name }} | {{ rows["bfd_per_link_rfc_7130"] | arista.avd.default(["-"]) | join(",") }} | {{ rows["e_lmi"] | arista.avd.default(["-"]) | join(",") }} | {{ rows["isis"] | arista.avd.default(["-"]) | join(",") }} | {{ rows["lacp"] | arista.avd.default(["-"]) | join(",") }} | {{ rows["lldp"] | arista.avd.default(["-"]) | join(",") }} | {{ rows["macsec"] | arista.avd.default(["-"]) | join(",") }} | {{ rows["pause"] | arista.avd.default(["-"]) | join(",") }} | {{ rows["stp"] | arista.avd.default(["-"]) | join(",") }} |
```

Comment:
@emilarista This line is tough to read. Can you set each value first in individual lines, then print them?

Reply:
Will do!